hub/visualizer/visualizer.py

```diff
@@ -92,23 +93,39 @@ def __del__(self):
 visualizer = _Visualizer()
 
 
+def _get_visualizer_backend_url():
+    if USE_LOCAL_HOST:
+        return "http://localhost:3000"
+    elif USE_DEV_ENVIRONMENT:
+        return "https://app.dev.activeloop.ai"
+    else:
+        return "https://app.activeloop.ai"
+
+
 def visualize(
-    storage: StorageProvider,
+    ds: Union[StorageProvider, str],
```

Comment:
would prefer a better argument name here

Reply:
What about source?

Reply:
yup that sounds better